docs/resources/obs_bucket.md

```diff
@@ -254,8 +257,13 @@ The `logging` object supports the following:
 
 * `target_bucket` - (Required, String) The name of the bucket that will receive the log objects. The acl policy of the
   target bucket should be `log-delivery-write`.
+
 * `target_prefix` - (Optional, String) To specify a key prefix for log objects.
 
+* `agency` - (Required, String) Specifies the IAM agency of OBS cloud service with the `PutObject` permission.
+  If default encryption is enabled for the current bucket, ensure that the agency has the `KMS Administrator` permission
```

Comment:
`the current bucket` or `the target bucket` ?

Reply:
It is `target bucket`. This issue has been fixed, please check it out.